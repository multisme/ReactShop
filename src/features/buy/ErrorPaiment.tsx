import React from "react";
import { useHistory } from "react-router-dom";

const ErrorPaiement = () => {
  const history = useHistory();

  setTimeout(() => {
    history.push("/pay");
  }, 5000);
  return (
    <div className="rectangle">
      <h1>Something went wrong with your paiement</h1>
      <h3>You will be redirected in no time</h3>
    </div>
  );
};

export default ErrorPaiement;
