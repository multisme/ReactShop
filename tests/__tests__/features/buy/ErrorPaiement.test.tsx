import ErrorPaiment from "features/buy/ErrorPaiment";
import { shallow } from "enzyme";

describe("ErrorPaiment", () => {
  it("renders", () => {
    const component = shallow(<ErrorPaiment />);
    expect(component).toMatchSnapshot();
  });
});
