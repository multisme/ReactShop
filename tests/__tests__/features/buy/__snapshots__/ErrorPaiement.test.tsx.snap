// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPaiment renders 1`] = `
<div
  className="rectangle"
>
  <h1>
    Something went wrong with your paiement
  </h1>
  <h3>
    You will be redirected in no time
  </h3>
</div>
`;
